docs/components/search/index.html

```diff
@@ -122,13 +170,27 @@ <h2 id="search-assistance">Search Assistance</h2>
 
       <footer>
         {% code 'html' %}
-          <hx-search
-            id="assisted-search"
-            onblur="hideAssistance()"
-            onfocus="showAssistance()"
-            value="item"
-          ></hx-search>
-          <hx-search-assistance relative-to="assisted-search">
+          <hx-search-control id="assisted-search-control">
+            <input
+              id="assisted-search"
+              type="search"
+              onblur="hideAssistance()"
+              onfocus="showAssistance()"
+            />
+            <button
+              type="button"
+              aria-label="Clear Search"
+              class="hxClear"
```

Comment:
nit: `class` before `aria-label` (to match order in other markup above)

Reply:
markup updated